Rest/Helper/FieldTypeValueHelper.php

```diff
@@ -0,0 +1,181 @@
+<?php
+/**
+ * This file is part of the EzSystemsRecommendationBundle package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Rest\Helper;
```

Comment:
It depends, sometimes you want to use composition of services.

I agree about the name and scope.